michbl/assignment3/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include "Dictionary.hpp"
+#include "Grid.hpp"
+#include "codeu_test_lib.h"
+
+int main() {
+    vector<char> g = {'A', 'A', 'R', 'T', 'C', 'D'};
+    Grid grid(g, 2, 3);
+
+    // test assignment example
+    vector<string> words = {"CAR", "CARD", "CART", "CAT"};
+    Dictionary dict1(words);
```

Comment:
Tip:
You can inline the words list:
```C++
Dictionary dict1({"CAR", "CARD", "CART", "CAT"});
```
It is really up to your personal preference :)